src/main/DeviceRow.jsx

```diff
@@ -62,6 +63,7 @@ const DeviceRow = ({ data, index, style }) => {
 
   const devicePrimary = useAttributePreference('devicePrimary', 'name');
   const deviceSecondary = useAttributePreference('deviceSecondary', '');
+  const deviceAttributeToShow = useAttributePreference('deviceShowAttribute', '');
```

Comment:
You use different names in different places:
- `deviceShowAttribute`
- `deviceAttributeToShow`
- `deviceAttribute`

Let's unify this, probably to `deviceShowAttribute`.